packages/react-qr/src/DisplayPayload.tsx

```diff
@@ -12,6 +12,7 @@ import QrDisplay from './Display';
 
 interface Props extends BaseProps {
   address: string;
+  cmd: Uint8Array;
```

Comment:
I still believe (as per the original comment), we should rather pass in a `number` here, since it is a single byte.